Formula/f/flexiblas-blis.rb

```diff
@@ -0,0 +1,97 @@
+class FlexiblasBlis < Formula
+  desc "BLIS runtime backend for FlexiBLAS"
+  homepage "https://www.mpi-magdeburg.mpg.de/projects/flexiblas"
+  url "https://csc.mpi-magdeburg.mpg.de/mpcsc/software/flexiblas/flexiblas-3.4.4.tar.xz"
+  sha256 "f3b4db7175f00434b1ad1464c0fd004f9b9ddf4ef8d78de5a75382a1f73a75dd"
+  license all_of: [
+    "LGPL-3.0-or-later",
+    "LGPL-2.1-or-later", # libcscutils/
```

Comment:
I think you can distribute this without `libcscutils/` license.

Backend is pretty much just one `.c` file and whatever headers it uses:

https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release/-/blob/master/src/CMakeLists.txt?ref_type=heads#L631
https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release/-/blob/master/src/CMakeLists.txt?ref_type=heads#L431
```cmake
    ADD_LIBRARY(flexiblas_${ivendor} SHARED ${BACKEND_CODE})
...
        BuildFindPackage(BlisSerial backends/generic/backend.c)
```

Trying locally, compiling the backend still works if I nuke the `build/libcscutils` directory